frontend/src/components/BoardCreationForm/BoardCreationForm.jsx

```diff
@@ -48,6 +53,13 @@ const BoardCreationForm = () => {
     setInputFileText(target.files[0].name);
   };
 
+  const handleMapClick = (cords) => {
+    setCords({
+      latitude: cords[1],
+      longitude: cords[0],
```

Comment:
Nie wiem, czy nie lepiej to rozczłonkować. 
cords[0] łatwo pomylić z cords[1]

Reply:
poprawiłem, żeby było czytelniej